tests/Unit/Capsule/DatabaseTest.php

```diff
@@ -33,8 +33,12 @@ public function test_selectCollection(): void
         self::assertEquals('testdb', $debugInfo['databaseName']);
     }
 
-    public function test_withOptions(): void
+    public function test_withOptions_using_mongodb_extension_lower_than_1_20_0(): void
```

Comment:
I wouldn't make those test mutually exclusive. 

As far as I see, the only thing that changes is the way to get the readPreference and make assertions on it. 
I would extract a method that get it which apply takes the extension version into account.

Reply:
Done! Thanks for the suggestion :)
#[Commit link](https://github.com/facile-it/mongodb-bundle/pull/187/commits/ef15b4de9fc361d2bc794b2c6a0bbcaa1dfe3922)